rfcs/text/0000-project-completion-stage.md

```diff
@@ -0,0 +1,74 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Project completion stage RFC
+
+## Summary
+
+This RFC proposes a process of releasing a contribution, or project, only after it has been completed by the DCP in order to maximize the value to the data consumer through reducing user confusion and minimizing errors in the final product.
+
+## Author(s)
+
+ [Jason Hilton](mailto:jahilton@stanford.edu)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Definitions
+
+A contribution, or project, is considered to be **version-complete** when the contributor confirms (by way of the wrangler) that they have provided all of the metadata and raw data that they plan to submit at the current time, the wrangler submits those to ingest, and the DCP has delivered every possible product, given current DCP capabilities.
```

Comment:
I want to second Tony's proposal, I think it better reflects current work and also means the text won't need to be as we move more of the submission process to be contributor-led rather than wrangler-led

Reply:
Thanks for the suggestion...though I would like to reserve the right to update the "submission" term after I wade through the discussion below